webapp-django/crashstats/settings/base.py

```diff
@@ -764,11 +696,11 @@ def path(*dirs):
     'secrets': {
         'postgresql': {
             'database_password': config(
-                'DATASERVICE_DATABASE_PASSWORD',
+                'DATABASE_PASSWORD',
                 'aPassword'
             ),
             'database_username': config(
-                'DATASERVICE_DATABASE_USERNAME',
+                'DATABASE_USERNAME',
```

Comment:
First of all, I don't know. 
There is a bit of a mismatch between the non-django work and the django world. It'd be nice to have a "common nomenclature" but not a must. There are some benefits of it being different too. 

The thing is, I think, thanks to `configman`, the value of this comes in from envconsul and by the fact that there's a config called `secrets.postgresql.database_username`.